ethereum/sidecar/server.go

```diff
@@ -45,6 +46,19 @@ var (
 	// enough.
 	cachedEventsLimit = 500000
 
+	// bridgeOutLookBackPeriod is the look-back period used when fetching
+	// AssetsUnlocked events from the Mezo chain. It defines how far back we
+	// look when searching for unattested events.
+	bridgeOutLookBackPeriod = 30 * 24 * time.Hour // ~1 month
```

Comment:
I realised i's a bit too late seeing that you have added the timestamps field in the AssetsUnlock events. But why use time instead of blockHeight for this? For some reason, the later seems more relevant to me when speaking of a blockchain. It's not a big deal though.